src/transformers/models/groupvit/modeling_tf_groupvit.py

```diff
@@ -63,6 +63,15 @@
             "It seems you have `tensorflow_probability` installed with the wrong tensorflow version."
             "Please try to reinstall it following the instructions here: https://github.com/tensorflow/probability."
         )
+else:
+    try:
+        import tensorflow_probability as tfp
+
+        # On the first call, check whether a compatible version of TensorFlow is installed
+        # TensorFlow Probability depends on a recent stable release of TensorFlow
+        _ = tfp.distributions.Normal(loc=0.0, scale=1.0)
+    except ImportError:
+        pass
```

Comment:
this look odd no?

Reply:
If this is about 

> tfp is not defined even if installed. FIXME @Arthur in a followup PR!"

similar to the test, OK for me for now, but maybe also add the same comment here.

Reply:
Yes even if tfp is installed it's not detecting it